ONSdigital/dp-frontend-dataset-controller
# Feature/add dataset template

### What

Added routing to direct pages to new dataset page template.
Added new mapper method to map zebedee data to frontend page model

### How to review

- Look at the code.
- Pull the code into your local environment with the corresponding dp-frontend-router and dp-frontend-renderer branches.
- Set 'DATASET_ENABLED' flag to true in environment config.
- Build and run the web stack.
- Navigate to an Old Workspace dataset versions page locally. e.g. /economy/inflationandpriceindices/datasets/consumerpriceinflation/current.
- See no changes in the page and it works exactly as before pulling the new code.

### Who can review

Anyone.
